### DISCUSSION TITLE
[p-sort] Não funciona com PO-UI  v15.15.0

### DISCUSSION THREAD

**Discussion by Josias-Ribeiro** [po-ui/po-angular#1891] [Tire suas dúvidas]
Boa tarde, estou com a seguinte questão, estou usando a versão do PO-UI  v15.15.0, mas a funcionalidade de sort da tabela não está funcionando. Vou mandar o trecho de código.

<po-table
  class="po-container"
  *ngIf="tableData"
  [p-actions-right]="true"
  [p-columns]="columns"
  [p-actions]="actions"
  [p-items]="tableData.data"
  [p-sort]="true"
></po-table>

e as colunas:
public columns: PoTableColumn[] = [
    {
      property: 'pensionName',
      label: this.translateService.instant('label.pension.table.pensions'),
      sortable: false
    },
    {
      property: 'startDate',
      label: this.translateService.instant('label.pension.table.startDate'),
      type: 'date',
      sortable: true,
    },
    {
      property: 'endDate',
      label: this.translateService.instant('label.pension.table.endDate'),
      type: 'date',
      sortable: true
    },
    {
      property: 'dateInclusion',
      label: this.translateService.instant('label.pension.table.dateInclusion'),
      type: 'date',
      sortable: true
    },
    {
      property: 'responsableUserName',
      label: this.translateService.instant('label.pension.table.userName'),
      sortable: true
    },
  ];
  
  Ele aparece aqueles icons (seta pra cima e seta pra baixo), mas ao clicar, o evento de sort não é disparado. Agradeço qualquer apoio.

**Comment by anliben** [comment]
Boa Tarde @Josias-Ribeiro, tudo bem?

Este item já foi solucionado nas versões posteriores 16.x

Attr.

**Reply by Josias-Ribeiro** [reply]
Mas é uma particularidade da versão 15.15.0?
